### PR TITLE
DOC-04 and updates for new navigation

### PR DESCRIPTION
**Pull request by YajB**
# What changed
https://deploy-preview-1261--logz-docs.netlify.app/user-guide/accounts/pci-compliance.html
- Updated introduction
- Added prerequisites
- updated images and navigation paths (removed "top menu")

https://docs.logz.io/user-guide/accounts/pci-compliance.html

----

## Pages to review

- LinkToPage1
- LinkToPage2

## Remaining work

- [ ] Technical review
- [ ] Copy Review
- [ ] Redirects: 

## Post launch

To be completed by the docs team upon merge:

- [ ] Teams to update with the new information:
- [ ] Replace original content on the support portal with 'this page has been moved to ...' - paste the URL
- [ ] Update these log shipping pages in the app: - paste the URL

### CODE REVIEW

File: _source/user-guide/accounts/pci.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ Minimize the risk of exposing sensitive customer data with Logz.io's log redacti
 automated sensitive-data scrubbing.
 
 Event logs are no exception when it comes to PCI compliance and Logz.io makes it simpler to prevent logging vulnerabilities and minimize the possibility of data breaches in your raw log files.
-Best practice is always to mask or redact sensitive information before writing it to the logs, but Logz.io PCI compliance offers an extra layer of protection that can scrutinize your logs for sensitive PPI such as customer credit card data. 
+Best practice is always to mask or redact sensitive information before writing it to the logs, but Logz.io PCI compliance offers an extra layer of protection: Based on the fields or patterns you can provide, we can mask or redact any sensitive PII (Personal Identifiable Information) that matches those fields or patterns, such as customer credit card data.
```
Comment (YajB) [inline _source/user-guide/accounts/pci.md:18]: @boofinka - your feedback is requested.

File: _source/user-guide/accounts/pci.md [hunk 2]
```diff
@@ -29,11 +29,24 @@ For more information on PCI compliance and DDS (Data Security Standard), see the
 
 <div class="tasklist">
 
+##### Prerequisites
+
+To insure that we redact the necessary details, your organization is responsible for providing Logz.io with the required cardholder information guidelines in writing, including but not limited to:
```
Comment (YajB) [inline _source/user-guide/accounts/pci.md:34]: @boofinka - your feedback requested.

File: _source/user-guide/accounts/pci.md (hunk 1 above)
Comment (boofinka) [reply _source/user-guide/accounts/pci.md:18]: Looks good.

**Review by boofinka — APPROVED** [review]
Looks good.

**Pull request merged**